fjohnson-instacartlite/app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,34 @@
+class OrdersController < ApplicationController
+
+    before_action :set_user
+
+    def index
+        @orders = @user.orders.all
```

Comment:
The `OrdersController` isn't the right place to be filtering by user.   You probably want a pattern that results in pages more like this:  https://admin.instacart.com/admin/customers/22438411/orders

This would be done by adding an `orders` method to the `UsersController` and nesting that.

Reply:
how come the list of the all the user's orders should go in an orders method in the users controller rather than in the index method of the orders controller?